src/components/Button/Button.stories.tsx

```diff
@@ -0,0 +1,80 @@
+import { Meta, StoryObj } from "@storybook/react";
+import Button from "./Button";
+
+const meta = {
+  title: "Components/Button",
+  parameters: {
+    docs: {
+      description: {
+        component: "A customizable button component.",
+      },
+    },
+  },
+  component: Button,
```

Comment:
The storybook is missing autodocs, if you add  `tags: ["autodocs"]` to the `meta` object then it will generate the autodocs